cpp/modules/deck.gl/core/src/views/map-view.h

```diff
@@ -33,12 +33,12 @@ class MapView : public View {
  public:
   using super = View;
 
-  static constexpr const char *getTypeName() { return "MapView"; }
-  auto getProperties() const -> const Properties * override;
+  static constexpr const char* getTypeName() { return "MapView"; }
```

Comment:
This is still changing? We still haven't locked down clang-format?

Reply:
This header wasn't part of the file list in cmake for some reason, figured that out once I included the framework in another project and it reported a few headers were missing